sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -65,7 +65,7 @@ import {
 } from "./explorer.props";
 import { previewReady } from "./preview";
 /* eslint-disable-next-line @typescript-eslint/no-var-requires */
-const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.png");
+const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.svg");
```

Comment:
you will need to add "svg" to the webpack.config.js for this site, change the file-loader test to:

```javascript
{
    test: /\.(svg|png|jpe?g|gif)$/i,
    use: {
        loader: "file-loader",
        options: {
            esModule: false,
        },
    },
},
```

Reply:
@janechu okay, will do.